2019-10-21T16:39:03.6920084+01:00;VSTestExecutionThread;Info;SpecFlow+Runner execution started
2019-10-21T16:39:03.6920084+01:00;VSTestExecutionThread;Info;SpecFlow+Runner 3.0.391 in Framework clr40 in x86 mode execution started
2019-10-21T16:39:03.6920084+01:00;VSTestExecutionThread;Info;TestAdapter Location: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\packages\SpecRun.Runner.3.0.391\tools\net45\TechTalk.SpecRun.VisualStudio.TestAdapter.dll
2019-10-21T16:39:03.8050111+01:00;VSTestExecutionThread;Info;SpecRun: running tests in C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\CSharpSpecFlowSelenium.dll
2019-10-21T16:39:03.8070103+01:00;VSTestExecutionThread;Info;Output folder configured to "C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\TestResults" (default)
2019-10-21T16:39:03.8080101+01:00;VSTestExecutionThread;Info;Profile: 
2019-10-21T16:39:03.8520100+01:00;VSTestExecutionThread;Info;Log file path: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\TestResults\Unnamed project__2019-10-21T163903.log
2019-10-21T16:39:03.8870101+01:00;VSTestExecutionThread;Info;SpecFlow+Runner Evaluation Mode: Please purchase at http://www.specflow.org/plus to remove test execution delay.
2019-10-21T16:39:03.8990461+01:00;Thread#14;Info;Checking license
2019-10-21T16:39:03.9420092+01:00;Thread#14;Info;Evaluation mode: test execution starts in 7 seconds... (this delay is removed in the registered version)
2019-10-21T16:39:10.9443585+01:00;Thread#14;Info;Evaluation delay finished.
2019-10-21T16:39:10.9573610+01:00;Thread#14;Info;License check completed
2019-10-21T16:39:10.9583617+01:00;Thread#14;Info;starting test run
2019-10-21T16:39:10.9903591+01:00;Thread#14;Info;Discovering target: Default
2019-10-21T16:39:10.9903591+01:00;Thread#14;Info;Discovering tests from assembly: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\CSharpSpecFlowSelenium.dll
2019-10-21T16:39:12.1393675+01:00:Appdomain Disposed
2019-10-21T16:39:12.1393675+01:00;Thread#14;Info;Discovered tests from assembly: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\CSharpSpecFlowSelenium.dll
2019-10-21T16:39:12.1413666+01:00:Test Suite Dump
2019-10-21T16:39:12.1433692+01:00:#0: Unnamed project/ - 
2019-10-21T16:39:12.1433692+01:00:#1: Default - 
2019-10-21T16:39:12.1433692+01:00:#2: CSharpSpecFlowSelenium - 
2019-10-21T16:39:12.1433692+01:00:#3: BBCLogin1 - 
2019-10-21T16:39:12.2253778+01:00:#4: Invalid password - TestAssembly:CSharpSpecFlowSelenium/Feature:BBCLogin1/Scenario:Invalid+password
2019-10-21T16:39:12.2253778+01:00:End: Test Suite Dump
2019-10-21T16:39:12.3193675+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T16:39:12.3263667+01:00:Thread#:Scheduler: testitem #4 from normal test queue 
2019-10-21T16:39:12.3273667+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-10-21T16:39:12.3623657+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T16:39:12.3663657+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-10-21T16:39:12.3663657+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-10-21T16:39:12.3663657+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-10-21T16:39:12.3733663+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-10-21T16:39:12.3753666+01:00:Thread#0:Scheduler: with testItemExecutionIndex 0
2019-10-21T16:39:12.3843683+01:00:Thread #0: creating new test thread executor for CSharpSpecFlowSelenium, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null (target: Default)
2019-10-21T16:39:12.3983667+01:00:Thread #0: Creating test assembly executor for execution model AppDomain
2019-10-21T16:39:12.4013705+01:00:Thread #0: Creating test assembly executor for execution model AppDomain
2019-10-21T16:39:12.4033678+01:00:Thread #0: using TechTalk.SpecRun.Framework.Execution.InProcess.InProcAppDomainTestAssemblyExecutorHost to execute tests
2019-10-21T16:39:12.7693706+01:00:Thread#0: Test #4/0 is started on thread #0
2019-10-21T16:39:13.2523733+01:00:Thread#0:T:-> Loading plugin C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\SpecRun.Runtime.SpecFlowPlugin.dll
2019-10-21T16:39:13.3033732+01:00:Thread#0:T:-> Using default config
2019-10-21T16:39:21.5946325+01:00:Thread#0:B:Given I am on the login page
2019-10-21T16:39:21.6636369+01:00:Thread#0:T:error: Object reference not set to an instance of an object.
2019-10-21T16:39:21.6766349+01:00:Thread#0:B:And I have enter a valid username
2019-10-21T16:39:21.6776357+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:21.6796360+01:00:Thread#0:B:And I have entered a invalid password
2019-10-21T16:39:21.6806360+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:21.6816416+01:00:Thread#0:B:When I press login
2019-10-21T16:39:21.6816416+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:21.6826521+01:00:Thread#0:B:Then I should see the appropriate error
2019-10-21T16:39:21.6836357+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:22.1606366+01:00:Thread#0:TestItem#4:Scheduler: reschedule tests for testitem #4
2019-10-21T16:39:22.1636377+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T16:39:22.1636377+01:00:Thread#:Scheduler: dequeuing a rescheduled testitem: #4
2019-10-21T16:39:22.1636377+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-10-21T16:39:22.1636377+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T16:39:22.1636377+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-10-21T16:39:22.1636377+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-10-21T16:39:22.1636377+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-10-21T16:39:22.1636377+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-10-21T16:39:22.1636377+01:00:Thread#0:Scheduler: with testItemExecutionIndex 1
2019-10-21T16:39:22.2186420+01:00:Thread#0: Test #4/0 is finished on thread #0 as Failed within 9.446269 seconds
2019-10-21T16:39:22.2206423+01:00:Thread#0: Test #4/1 is started on thread #0
2019-10-21T16:39:31.1799248+01:00:Thread#0:B:Given I am on the login page
2019-10-21T16:39:31.1839568+01:00:Thread#0:T:error: Object reference not set to an instance of an object.
2019-10-21T16:39:31.1839568+01:00:Thread#0:B:And I have enter a valid username
2019-10-21T16:39:31.1839568+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:31.1849255+01:00:Thread#0:B:And I have entered a invalid password
2019-10-21T16:39:31.1849255+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:31.1849255+01:00:Thread#0:B:When I press login
2019-10-21T16:39:31.1849255+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:31.1849255+01:00:Thread#0:B:Then I should see the appropriate error
2019-10-21T16:39:31.1859252+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:31.1869254+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T16:39:31.1869254+01:00:Thread#:Scheduler: dequeuing a rescheduled testitem: #4
2019-10-21T16:39:31.1869254+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-10-21T16:39:31.1869254+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T16:39:31.1869254+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-10-21T16:39:31.1869254+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-10-21T16:39:31.1869254+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-10-21T16:39:31.1869254+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-10-21T16:39:31.1869254+01:00:Thread#0:Scheduler: with testItemExecutionIndex 2
2019-10-21T16:39:31.1879242+01:00:Thread#0: Test #4/1 is finished on thread #0 as Failed within 8.9672819 seconds
2019-10-21T16:39:31.1889254+01:00:Thread#0: Test #4/2 is started on thread #0
2019-10-21T16:39:39.5620414+01:00:Thread#0:B:Given I am on the login page
2019-10-21T16:39:39.5680309+01:00:Thread#0:T:error: Object reference not set to an instance of an object.
2019-10-21T16:39:39.5680309+01:00:Thread#0:B:And I have enter a valid username
2019-10-21T16:39:39.5690305+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:39.5690305+01:00:Thread#0:B:And I have entered a invalid password
2019-10-21T16:39:39.5690305+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:39.5690305+01:00:Thread#0:B:When I press login
2019-10-21T16:39:39.5700292+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:39.5700292+01:00:Thread#0:B:Then I should see the appropriate error
2019-10-21T16:39:39.5700292+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:39:39.5710307+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T16:39:39.5710307+01:00:Thread#:Scheduler: at the end of normal test queue
2019-10-21T16:39:39.5710307+01:00:Thread#0:Scheduler: enumerator for thread is at end
2019-10-21T16:39:39.5710307+01:00:Thread#0:Scheduler: No more tests for thread
2019-10-21T16:39:39.5740308+01:00:Thread#0: Test #4/2 is finished on thread #0 as Failed within 8.3851054 seconds
2019-10-21T16:39:39.6230295+01:00:Appdomain Disposed
2019-10-21T16:39:39.6250311+01:00:Trying to cancel thread
2019-10-21T16:39:39.6250311+01:00:Thread cancelled
2019-10-21T16:39:39.6380332+01:00:Thread#-1:Scheduler: get next test for thread -1
2019-10-21T16:39:39.6380332+01:00:Thread#:Scheduler: at the end of normal test queue
2019-10-21T16:39:39.6380332+01:00:Thread#-1:Scheduler: enumerator for thread is at end
2019-10-21T16:39:39.6570303+01:00;Thread#14;Info;test run finished
2019-10-21T16:39:39.6580303+01:00;Thread#14;Info;publishing test results
2019-10-21T16:39:39.6760292+01:00;Thread#14;Info;test results published
2019-10-21T16:39:39.6760292+01:00;Thread#14;Info;generating reports
2019-10-21T16:39:40.0550297+01:00;Thread#14;Info;Starting external report generation process
2019-10-21T16:39:45.3070567+01:00;Thread#14;Info;Successfully generated reports.
2019-10-21T16:39:45.3130926+01:00;Thread#14;Info;Successfully generated reports
2019-10-21T16:39:45.3310588+01:00:Total: 1 (test executions: 3)
2019-10-21T16:39:45.3340578+01:00:Succeeded: 0
2019-10-21T16:39:45.3350569+01:00:Ignored: 0
2019-10-21T16:39:45.3370590+01:00:Pending: 0
2019-10-21T16:39:45.3370590+01:00:Skipped: 0
2019-10-21T16:39:45.3370590+01:00:Failed: 1
2019-10-21T16:39:45.3650602+01:00;VSTestExecutionThread;Info;Adding attachments to VSTest
2019-10-21T16:39:45.3670599+01:00;VSTestExecutionThread;Info;adding Test Execution Report 'C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\TestResults\Unnamed project__2019-10-21T163903.html'
========== End of log file ==========
